.husky/pre-push

```diff
@@ -0,0 +1,22 @@
+# Check for unstaged changes
+if ! git diff-index --quiet HEAD --; then
+  echo "uou have unstaged or uncommitted files. commit, stash or discard your changes before pushing."
```

Comment:
typo `uou`

Reply:
before `before commiting` instead of `before pushing`?

Reply:
why before committing? 
it asks to do smthg before push with uncommitted or unstaged changes
i can simplify to 'Please commit or revert your changes before push', wdyt?

Reply:
is this a commit hook or git push hook?